hbase_regionserver/metadata.csv

```diff
@@ -7,9 +7,9 @@ hbase.regionserver.ipc.total_call_time.mean,gauge,"",millisecond,"","total call
 hbase.regionserver.ipc.total_call_time.median,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.ipc.total_call_time.percentile.99,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.regions.num_regions,gauge,"","","",Number of regions in the metrics system,0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_ops,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_batches,gauge,"","","","",0,hbase_regionserver,""
+hbase.regionserver.replication.sink.applied_ops,gauge,"","","",Number of WAL entries applied on replication sink.,0,hbase_regionserver,""
+hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","",Time lag of last applied WAL entry between master and slave in milliseconds.,0,hbase_regionserver,""
```

Comment:
@iwasakims  Does metrics name including "age" induce that unit of the metrics will be `millisecond`!?  If so, we could add this rule to unit detection in [`_gen.rb`](https://github.com/everpeace/integrations-extras/blob/hbase/hbase_regionserver/_gen.rb#L39-L51).